src/js/nettest.js

```diff
@@ -66,10 +66,9 @@ NetworkTest.prototype = {
           newUrls.push(uri);
         } else if (
           uri.indexOf('?transport=') === -1 && uri.startsWith('turn')) {
-          newUrls.push(uri + '?' + transport);
+            newUrls.push(uri + '?' + transport);
         }
       }
-      iceServer.urls = newUrls;
```

Comment:
Removing this line doesn't makes sense. The whole purpose to construct newUrls is to assign it somewhere.